### PR TITLE
DBZ-78 add example for topic auto creation

### PR DESCRIPTION
**Pull request by rk3rn3r**
related to: https://issues.redhat.com/browse/DBZ-78 and https://github.com/debezium/debezium.github.io/pull/490

### CODE REVIEW

File: topic-auto-create/README.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        "topic.creation.productlog.compression.type": "producer",
+        "topic.creation.productlog.retention.ms": 7776000000
+    }
+}
+```
+
+### Sending connector config
+
+```bash
+$ curl -X POST -H "Accept: application/json" -H "Content-Type: application/json" -d @connector.json "http://localhost:8083/connectors" | json_pp
```
Comment (gunnarmorling) [inline topic-auto-create/README.md:75]: Let's use PUT, as it's idempotent and generally preferred in our examples (I know it's not done yet consistently).

File: topic-auto-create/README.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+``` 
+
+### Validating the result
+
+```bash
+## Login into the Debezium container
+$ docker exec -it topic-auto-create_kafka_1 bash
+
+## Check the orders topic, if the `default` group config was applied
+$ /kafka/bin/kafka-topics.sh --bootstrap-server $HOSTNAME:9092 --describe --topic dbserver1.inventory.orders
```
Comment (gunnarmorling) [inline topic-auto-create/README.md:118]: I'd probably have used the Debezium tooling image and kafkacat, but it's fine that way, too.

**Review by gunnarmorling — COMMENTED** [review]
LGTM overall. Minor comments inline, feel free to merge yourself once done. Thx!

Could you also log a Jira issue for adding a README.md file in the repo root, listing all examples. Thx!

File: topic-auto-create/README.md (hunk 1 above)
Comment (rk3rn3r) [reply topic-auto-create/README.md:75]: I never use PUT. With POST and having the connector name inside the JSON file is much more reliable and idempotent compared to the PUT way from my experiences when you have your connector config checked-in to VCS.

Comment (rk3rn3r) [reply topic-auto-create/README.md:75]: Also for the examples I think POST-ing the full config with connector name in JSON makes it "Luser-proof".

Comment (rk3rn3r) [reply topic-auto-create/README.md]: Done. But let me challenge you on that topic later!

**Comment by rk3rn3r** [comment]
created https://issues.redhat.com/browse/DBZ-2535 for extending the root README.md with a list of all examples

**Pull request merged**